bpf/Makefile

```diff
@@ -28,6 +28,7 @@ all: $(BPF) $(TARGET)
 
 check:
 	@$(ECHO_CHECK) bpf/*.c
+	$(QUIET) spatch --sp-file ../contrib/coccinelle/null.cocci --dir . --include-headers
```

Comment:
$(QUIET) usually tries to silence non-error output such as the "HANDLING..." lines. Is there a --quiet argument or something we could use here?

Reply:
There is, but it doesn't oppress these lines. Adding `2> /dev/null` would oppress them though.

Reply:
Otoh, it's probably also okay to verify which files were checked. I don't mind either way.

Reply:
I like having only the errors printed, and the error output seems to provide the filename so I don't think the `HANDLING` lines tell us anything. But I can also follow up on this later.

Reply:
Additionally, I think we could add further useful cocci scripts to contrib and split up the `make check` in subcommands such that we have one for sparse, one focussed on cocci, and one on clang analyzer.